core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -101,6 +111,28 @@ oppia.directive('tutorCard', [
             }
           };
 
+          $scope.consumeSolution = function() {
+            playerTranscriptService.addNewInput(
+              'Please show me the answer.', true);
+            var solution = SolutionManagerService.consumeSolution();
+            var interactionId = stateInteractionIdService.savedMemento;
+            var answer = '';
+            if (interactionId === 'MultipleChoiceInput') {
+              answer = (
+                responsesService.getAnswerChoices()[solution.correctAnswer]
+                .label);
+            } else if (interactionId === 'MathExpressionInput') {
+              answer = solution.correctAnswer.latex;
+            } else {
+              answer = solution.correctAnswer;
+            }
+            var answerIsExclusive = solution.answerIsExclusive ? 'Only' : 'One';
```

Comment:
Can you change it from "Only..." to "The only..."? It sounds awkward with the former.